twiggy/tests/all/diff_tests.rs

```diff
@@ -0,0 +1,82 @@
+extern crate colored;
+extern crate diff;
+
+use std::process::Command;
+
+use colored::Colorize;
+
+use slurp;
```

Comment:
If we do the imports *inside* the function that the test macro expands into, then every test module won't have to have the same seemingly-unused imports.